sklearn/utils/estimator_checks.py

```diff
@@ -211,6 +211,12 @@ def _boston_subset(n_samples=200):
         BOSTON = X, y
     return BOSTON
 
+def get_kwargs(estimator_class):
```

Comment:
ping @amueller @ogrisel  @MechCoder 
Some tests warned because the SVM object wasn't initialized with the correct arguments. Is there a better way to fix this ?